SWIG/inflation.i

```diff
@@ -264,6 +264,14 @@ struct CPI {
     enum InterpolationType { AsIndex, Flat, Linear };
 };
 
+%inline %{
+    Real laggedFixing(const ext::shared_ptr<ZeroInflationIndex>& index,
+                      const Date& date,
+                      const Period& observationLag,
+                      CPI::InterpolationType interpolationType) {
+        return QuantLib::CPI::laggedFixing(index, date, observationLag, interpolationType);
+    }
+%}
```

Comment:
There's no need for a wrapper function — you can declare the static `laggedFixing` method inside the `CPI` struct above.

Reply:
Ok, thanks. I'll do that.